daml-assistant/src/DAML/Assistant/Install/Github.hs

```diff
@@ -0,0 +1,112 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Discover releases from the digital-asset/daml github.
+module DAML.Assistant.Install.Github
+    where
+
+import DAML.Assistant.Types
+import DAML.Assistant.Util
+import Data.Aeson
+import Network.HTTP.Simple
+import Control.Exception.Safe
+import Control.Monad
+import Data.List
+import Data.Either.Extra
+import qualified System.Info
+import qualified Data.Text as T
+
+newtype Tag = Tag Text deriving (Eq, Show, FromJSON)
```

Comment:
Can we have an example tag name as document. Is it v1.0, 1.0, or is tag something totally different?

Reply:
:+1: